apps/emqx_modules/test/emqx_telemetry_SUITE.erl

```diff
@@ -28,14 +28,53 @@
 all() -> emqx_common_test_helpers:all(?MODULE).
 
 init_per_suite(Config) ->
-    ok = mria:start(),
-    ok = emqx_telemetry:mnesia(boot),
+    snabbkaffe:fix_ct_logging(),
     emqx_common_test_helpers:start_apps([emqx_modules]),
     Config.
 
 end_per_suite(_Config) ->
     emqx_common_test_helpers:stop_apps([emqx_modules]).
 
+init_per_testcase(t_get_telemetry, Config) ->
+    DataDir = ?config(data_dir, Config),
+    TestPID = self(),
+    ok = meck:new(httpc, [non_strict, passthrough, no_history, no_link]),
+    ok = meck:expect(httpc, request, fun(Method, URL, Headers, Body) ->
+                                             TestPID ! {request, Method, URL, Headers, Body}
+                                     end),
+    ok = meck:new(emqx_telemetry, [non_strict, passthrough, no_history, no_link]),
+    ok = meck:expect(emqx_telemetry, read_raw_build_info,
+                     fun() ->
+                             {ok, Path} = file:read_link(filename:join([DataDir, "BUILD_INFO"])),
+                             {ok, Template} = file:read_file(Path),
+                             Vars0 = [ {build_info_arch, "arch"}
+                                    , {build_info_wordsize, "64"}
```

Comment:
Nit: indentation

Reply:
I'll merge it like this and follow up with the formatting of `emqx_modules`, which is not formatted yet.